src/liballoc/raw_vec.rs

```diff
@@ -579,11 +578,10 @@ impl<T> RawVec<T, Global> {
             "`len` must be smaller than or equal to `self.capacity()`"
         );
 
+        let me = ManuallyDrop::new(self);
         // NOTE: not calling `capacity()` here; actually using the real `cap` field!
```

Comment:
Any idea what is up with that comment? `cap` is not used here, before not after your change.

Reply:
Seems that this happened recently in #70362, specifically in commit 2526accdd35c564eee80b6453a0b4965e6a76afd.

Reply:
Looks like just an outdated comment then. Do you want to remove it?

Reply:
This PR is already r+-and-rollup-ed, so I'll remove the comment once the PR is merged to avoid merge issues.

Reply:
Ah I'll just add it to https://github.com/rust-lang/rust/pull/70776 then

Reply:
Yes, the comment is outdated and I forgot to remove it.